rotkehlchen/data/all_assets.json

```diff
@@ -9414,6 +9414,14 @@
         "symbol": "ULTC",
         "type": "own chain"
     },
+    "UMA": {
+        "ethereum_address": "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828",
+        "ethereum_token_decimals": 18,
+        "name": "UMA Voting Token v1",
+        "started": 1590364800,
```

Comment:
This should be the address from which we believe the token started to exist. For UMA this seems to be: `1578581061` according to the contract deployment transaction: https://etherscan.io/tx/0x071e4641c466453e04cd864097971534d81a9a8a9e61f21219e984df3cc48480

Can you change that?